telegram/utils/request.py

```diff
@@ -135,12 +135,14 @@ def _parse(json_data):
     except ValueError:
         raise TelegramError('Invalid server response')
 
-    if not data.get('ok') and data.get('description'):
-        return data['description']
+    if not data.get('ok'):
+        if data.get('parameters') and data['parameters'].get('migrate_to_chat_id'):
```

Comment:
please save `data.get('parameters')` into a variable, so you won't need to get it 3 times.